pytensor/link/numba/dispatch/basic.py

```diff
@@ -65,7 +66,8 @@ def numba_njit(*args, **kwargs):
     # Supress caching warnings
     warnings.filterwarnings(
         "ignore",
-        message='Cannot cache compiled function "numba_funcified_fgraph" as it uses dynamic globals',
+        # FIXME: Specific messages are not working
+        # message='Cannot cache compiled function "store_core_outputs" as it uses dynamic globals (such as ctypes pointers and large global arrays)',
```

Comment:
I cannot get the warning suppressed with any message whatsoever. The relevant test is 

https://github.com/pymc-devs/pytensor/blob/1e96b8942bc42fd48af50e138c5b2458083c6091/tests/link/numba/test_basic.py#L1071-L1078

Reply:
Wasn't there a pymc issue that we were setting warning levels wrong in a way they couldn't be ignored? I can't find the issue though.

Reply:
I think that's different. That was about the logging module, not warnings.